partner_firstname/models/res_partner.py

```diff
@@ -65,6 +65,7 @@ def copy(self, default=None):
     @api.model
     def default_get(self, fields_list):
         """Invert name when getting default values."""
+        fields_list.append("name")
```

Comment:
IMO, the proper fix is to do the stuff only if `name` if `fields_list`, not to force always to include that field in the results.

Reply:
Hi @pedrobaeza, I'm not sure to understand what do you propose, but I added a condition to add `name` only if `firstname` or `lastname` is in `fields_list`.
Let me know if there is a better solution

Reply:
OK, that can work.

Reply:
Pr for FIX https://github.com/OCA/partner-contact/pull/1793